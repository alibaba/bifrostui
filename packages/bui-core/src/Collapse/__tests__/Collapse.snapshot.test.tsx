import { snapshotTest } from 'testing';

describe('Collapse snapshot', () => {
  snapshotTest('Collapse');
});
