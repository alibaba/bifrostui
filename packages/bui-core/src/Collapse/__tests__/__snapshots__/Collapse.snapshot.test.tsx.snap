// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Collapse snapshot > Collapse demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "wrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <div
    className="bui-switch bui-switch-medium bui-switch-primary"
  >
    <input
      checked={false}
      className="bui-switch-input"
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
  </div>
  <div
    className="bui-collapse"
    style={
      {
        "WebKitWidth": "0px",
        "WebkitTransition": "width 1500ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "transition": "width 1500ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "width": "0px",
      }
    }
  >
    <div
      className="bui-card"
      style={
        {
          "background": "#CCCCCC",
          "height": "100px",
          "width": "100px",
        }
      }
    />
  </div>
</div>
`;

exports[`Collapse snapshot > Collapse demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "wrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <div
    className="bui-switch bui-switch-medium bui-switch-primary bui-switch-checked"
  >
    <input
      checked={true}
      className="bui-switch-input"
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
  </div>
  <div
    className="bui-collapse"
    style={
      {
        "WebKitHeight": "fit-content",
        "WebkitTransition": "height 1500ms cubic-bezier(0.0, 0, 0.2, 1) 0ms",
        "height": "fit-content",
        "transition": "height 1500ms cubic-bezier(0.0, 0, 0.2, 1) 0ms",
      }
    }
  >
    <div
      className="bui-card"
      style={
        {
          "background": "#CCCCCC",
          "height": "100px",
          "width": "100px",
        }
      }
    />
  </div>
</div>
`;

exports[`Collapse snapshot > Collapse demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "wrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <div
    className="bui-switch bui-switch-medium bui-switch-primary bui-switch-checked"
  >
    <input
      checked={true}
      className="bui-switch-input"
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
  </div>
  <div
    className="bui-collapse"
    style={
      {
        "WebKitHeight": "fit-content",
        "WebkitTransition": "height 1500ms cubic-bezier(0.0, 0, 0.2, 1) 0ms",
        "height": "fit-content",
        "transition": "height 1500ms cubic-bezier(0.0, 0, 0.2, 1) 0ms",
      }
    }
  >
    <div
      className="bui-card"
      style={
        {
          "background": "#CCCCCC",
          "height": "100px",
          "width": "100px",
        }
      }
    />
  </div>
</div>
`;

exports[`Collapse snapshot Collapse demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "wrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <div
    className="bui-switch bui-switch-medium bui-switch-primary"
  >
    <input
      checked={false}
      className="bui-switch-input"
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
  </div>
  <div
    className="bui-collapse"
    style={
      Object {
        "WebKitWidth": "0px",
        "WebkitTransition": "width 1500ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "transition": "width 1500ms cubic-bezier(0.4, 0, 0.6, 1) 0ms",
        "width": "0px",
      }
    }
  >
    <div
      className="bui-card"
      style={
        Object {
          "background": "#CCCCCC",
          "height": "100px",
          "width": "100px",
        }
      }
    />
  </div>
</div>
`;

exports[`Collapse snapshot Collapse demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "wrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <div
    className="bui-switch bui-switch-medium bui-switch-primary bui-switch-checked"
  >
    <input
      checked={true}
      className="bui-switch-input"
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
  </div>
  <div
    className="bui-collapse"
    style={
      Object {
        "WebKitHeight": "fit-content",
        "WebkitTransition": "height 1500ms cubic-bezier(0.0, 0, 0.2, 1) 0ms",
        "height": "fit-content",
        "transition": "height 1500ms cubic-bezier(0.0, 0, 0.2, 1) 0ms",
      }
    }
  >
    <div
      className="bui-card"
      style={
        Object {
          "background": "#CCCCCC",
          "height": "100px",
          "width": "100px",
        }
      }
    />
  </div>
</div>
`;

exports[`Collapse snapshot Collapse demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "wrap",
      "--justify-content": "center",
      "--spacing": "8px",
    }
  }
>
  <div
    className="bui-switch bui-switch-medium bui-switch-primary bui-switch-checked"
  >
    <input
      checked={true}
      className="bui-switch-input"
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
  </div>
  <div
    className="bui-collapse"
    style={
      Object {
        "WebKitHeight": "fit-content",
        "WebkitTransition": "height 1500ms cubic-bezier(0.0, 0, 0.2, 1) 0ms",
        "height": "fit-content",
        "transition": "height 1500ms cubic-bezier(0.0, 0, 0.2, 1) 0ms",
      }
    }
  >
    <div
      className="bui-card"
      style={
        Object {
          "background": "#CCCCCC",
          "height": "100px",
          "width": "100px",
        }
      }
    />
  </div>
</div>
`;
